compiler/tests-spec/testData/diagnostics/notLinked/contracts/analysis/controlFlow/initialization/pos/3.fir.kt

```diff
@@ -155,7 +155,7 @@ fun case_10() {
     val x: Int
     funWithExactlyOnceCallsInPlace outer@ {
         funWithAtLeastOnceCallsInPlace {
-            x = 42
+            <!VAL_REASSIGNMENT!>x<!> = 42
```

Comment:
This is the only one that differs from FE 1.0 behavior, but I can't tell whether it's a false alarm or not. Due to the labeled return below, this lambda exits to outer, and thus won't be invoked multiple times. However, as the function name indicates, I guess the contract for the caller is that the lambda can be invoked _at least_ once, hence the event occurrence count for this variable would be `AT_LEAST_ONCE`, which also means this can be revisited.

I don't think there is anything we can do regarding CFG/contract here. cc @demiurg906